src/index.ts

```diff
@@ -44,6 +45,7 @@ export {
   WithLabel,
   WithValidationError,
   useModal,
+  useIsMountedRef,
```

Comment:
maybe i dont understand, but where do we use it?

Reply:
aa nevermind. is the components library. 🤦  but i do agree with @lkappeler i think we should reconsider the place for this. maybe we can have a discussion about this in an RFC?

Reply:
it was my first reaction as well, but in the end we're sharing hooks already now. they're only gonna be used in components, so this feels like the right place to me. I would not want an extra package just for hooks and then end up consuming both packages in most projects anyway :)

(just my 2 cents. I'm open to exploring alternative ways)

Reply:
@ralucacril I used it here https://github.com/carforyou/carforyou-dealerhub-web/pull/880

The location seems odd but there is already the `useModal` hook inside the component package. I'm fine with removing it but eventually, we will use it in listings and dealerhub